src/main/java/core/basesyntax/DateConvertor.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class DateConvertor {
```

Comment:
separate class for converting - cool!

Reply:
I hope this isn't a joke

Reply:
Not at all!
Looks like the first principle of SOLID

Reply:
Yeah, i understood a little about solids, although I am not sure that I fully understood. Especially when it comes to Spring..